flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -146,6 +144,12 @@ public class ExecutionConfig implements Serializable, Archiveable<ArchivedExecut
      */
     private final Configuration configuration;
 
+    private final SerializerConfig serializerConfig;
+
+    public SerializerConfig getSerializerConfig() {
+        return serializerConfig;
```

Comment:
Is it appropriate for this object to be part of the public API? I couldn't find this change mentioned in the FLIP.
If not, I prefer to annotate this method by Internal. If yes, the change should be reflected in the "Public Propose" section of the FLIP documentation.

Reply:
Thanks for pointing it out. I'll annotate it @Internal